conf/models.yml

```diff
@@ -21,3 +21,18 @@ model: &least_squares
 
 model: &xgboost
     type: xgboost
+
+model: &averaging
+    type: averaging
+
+model: &decision_tree
+    type: decision_tree
+
+model: &mlp
+    type: mlp
```

Comment:
we should sync this up with [what's in Garg et al.](https://github.com/dtsip/in-context-learning/blob/main/src/models.py#L51-L64) the defaults are much smaller than that to make CI run faster